src/control-plane/backend/lambda/api/service/quicksight/reporting-utils.ts

```diff
@@ -387,7 +388,7 @@ export const createDataSet = async (quickSight: QuickSight, awsAccountId: string
       AwsAccountId: awsAccountId,
       DataSetId: datasetId,
       Name: datasetId,
-      ImportMode: props.importMode,
+      ImportMode: props.useSpice === 'yes' ? DataSetImportMode.SPICE : DataSetImportMode.DIRECT_QUERY,
```

Comment:
defining `useSpice` as boolean is simpler

Reply:
Its value comes from a CFN parameter, defining it as a boolean isn't very convenient.